utils.js

```diff
@@ -53,6 +59,24 @@ function post(url, jar, form) {
   return request(op).then(function(res) {return res[0];});
 }
 
+function postFormData(url, jar, form, qs) {
```

Comment:
The ```Content-Type``` is multipart/form-data and it's using ```formData``` instead of ```form``` option, hence the function name.